appscale/tools/remote_helper.py

```diff
@@ -825,7 +841,7 @@ def terminate_cloud_infrastructure(cls, keyname, is_verbose):
       is_verbose: A bool that indicates if we should print the commands executed
         to stdout.
     """
-    AppScaleLogger.log("About to terminate instances spawned with keyname {0}"
+    AppScaleLogger.log("Will terminate deployment and instances spawned with keyname {0}"
```

Comment:
Since it seems like the reason for this message is to give the user an opportunity to cancel, we may want to mention that they can Ctrl-C.